modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/clientExtensionCellRenderer.spec.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../../../fixtures/featureFlagsTest';
+import {loginTest} from '../../../../fixtures/loginTest';
+import getRandomString from '../../../../utils/getRandomString';
+import {dataSetManagerApiHelpersTest} from '../../fixtures/dataSetManagerApiHelpersTest';
+import clickActionInRow from '../../utils/clickActionInRow';
+import saveFromModal from '../../utils/saveFromModal';
+import {dataSetManagerSetupTest} from './fixtures/dataSetManagerSetupTest';
+import {visualizationModesPageTest} from './fixtures/visualizationModesPageTest';
+
+export const test = mergeTests(
+	dataSetManagerApiHelpersTest,
+	featureFlagsTest({
+		'LPS-164563': true,
+	}),
+	visualizationModesPageTest,
+	loginTest(),
+	dataSetManagerSetupTest
+);
+
+let dataSetERC: string;
+
+const dataSetLabel: string = getRandomString();
+
+test.beforeEach(async ({dataSetManagerApiHelpers}) => {
+	dataSetERC = getRandomString();
+
+	await dataSetManagerApiHelpers.createDataSet({
+		erc: dataSetERC,
+		label: dataSetLabel,
+	});
+});
+
+test.afterEach(async ({dataSetManagerApiHelpers}) => {
+	await dataSetManagerApiHelpers.deleteDataSet({
+		erc: dataSetERC,
+	});
+});
+
+test('Assert the CellRenderer is displayed', async ({
+	page,
+	visualizationModesPage,
+}) => {
+	const sampleScalarField = 'id';
+	const sampleObjectField = 'fdsViewFDSFieldRelationship';
```

Comment:
With https://github.com/brianchandotcom/liferay-portal/pull/154813 merged, this will now break. We changed relationship names. This looks like `dataSetToDataSetTableSections`.